### ISSUE TITLE
Track when a user doesnt pick up their laundry

### ISSUE THREAD

**Issue by vidia** [Purdue-ACM-SIGAPP/PurdueLaundry#78]
We know what machine a user is using if they set a timer. We can alert the user if they havent taken out their laundry after a certain period of time ("End of Cycle" state)

We could also do this intelligently and only ask the user for to move their stuff if the other machines are full.

**Comment by hughesjeff** [comment]
So, this would probably actually anger me if I were a user. I don't like the concept of an app telling me what to do in this case. It would be nice to have stats on it though :P

**Comment by vidia** [comment]
I agree in most cases. But it would be nice to at least track when a specific user of the app leaves their laundry in too long when the rest of the room is packed. 

Then we can decide if it would be a good idea to send a "Did you forget to remove your laundry" notification. Though I understand the potential for annoyance here.

**Comment by vieck** [comment]
I've gotten another request for this in person so we should bump this up in priority
